src/common/TrackedOp.h

```diff
@@ -170,7 +173,9 @@ class TrackedOp {
     seq(0),
     warn_interval_multiplier(1),
     is_tracked(false)
-  { }
+  {
+    tid = 0;
```

Comment:
More generally, I don't see a good reason to add a tid to TrackedOp, I'd rather you just removed this bit.